tutorials/nlp/Dialogue.ipynb

```diff
@@ -1,717 +1,2338 @@
+    "    transport type (label_id: 52)                           97.06      94.29      95.65         35\n",
+    "    weather descriptor (label_id: 53)                       90.91      66.67      76.92         15\n",
+    "    other (label_id: 54)                                    96.59      96.32      96.46       3291\n",
+    "    -------------------\n",
+    "    micro avg                                               92.53      92.53      92.53       4523\n",
+    "    macro avg                                               76.34      77.14      74.44       4523\n",
+    "    weighted avg                                            92.99      92.53      92.56       4523\n",
+    "\n",
+    "┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓\n",
+    "┃           Test metric            ┃           DataLoader 0           ┃\n",
+    "┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩\n",
+    "│           bio_slot_f1            │        94.04096984863281         │\n",
+    "│        bio_slot_precision        │        94.04096984863281         │\n",
+    "│         bio_slot_recall          │        94.04096984863281         │\n",
+    "│         overall_slot_f1          │        92.52709197998047         │\n",
+    "│      overall_slot_precision      │        92.52708435058594         │\n",
+    "│       overall_slot_recall        │        92.52708435058594         │\n",
+    "│        slot_similarity_f1        │              91.875              │\n",
+    "│    slot_similarity_precision     │              91.875              │\n",
+    "│      slot_similarity_recall      │              91.875              │\n",
+    "│         unified_slot_f1          │        80.28195816548065         │\n",
+    "│ unified_slot_joint_goal_accuracy │         71.2624584717608         │\n",
+    "│      unified_slot_precision      │        79.27740863787376         │\n",
+    "│       unified_slot_recall        │        81.31229235880399         │\n",
+    "│             val_loss             │        0.6432915925979614        │\n",
+    "└──────────────────────────────────┴──────────────────────────────────┘\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4.5 (Optional) Download CoNLL-2003 dataset and convert to NeMo format\n",
+    "\n",
+    "\n",
+    "The shared task of CoNLL-2003 concerns language-independent named entity recognition. We will concentrate on four types of named entities: persons, locations, organizations and names of miscellaneous entities that do not belong to the previous three groups.\n",
+    "\n",
+    "\n",
+    "For more details see https://www.clips.uantwerpen.be/conll2003/ner/ and https://www.aclweb.org/anthology/W03-0419\n",
+    "\n",
+    "######################################################################\n",
+    "\n",
+    "Licensing Information\n",
```

Comment:
Is there a reason why this entire Licensing agreement needs to be here? It looks a little weird because you are downloading from a separate website (deepai).